File: ljh/java/src/level1/SortInDescendingOrder.java
package level1;

import java.util.Arrays;

/*
 * 문자열 내림차순으로 정렬하기
 */

public class SortInDescendingOrder {
	public String solution(String s) {
		int[] nums = new int[s.length()];
		String answer = "";
		
		for(int i=0; i<s.length(); i++) {
			nums[i] = -1 * s.charAt(i);
		}
		Arrays.sort(nums);
		for(int j=0; j<nums.length; j++) {
			 answer += (char)(nums[j] * -1);
		}
		
		return answer;
	}
	
}

File: ljh/javascript/level1/SortInDescendingOrder.js
// 문자열 내림차순으로 정렬하기(12/18 - 12/24)
export default class SortInDescendingOrder {
    solution = (s) => {
        let answer = [];
        for(let c of s) answer.push(c.charCodeAt());
        answer.sort((a, b) => b - a);
        return answer.map((value) => String.fromCharCode(value)).join('');
    }
}